packages/editor-sdk/src/components/Form/RecordEditor.tsx

```diff
@@ -60,16 +62,16 @@ const RowItem = (props: RowItemProps) => {
     }),
     []
   );
-  const valueSpec = useMemo(() => spec?.properties?.[rowKey], [spec, rowKey]);
+  const valueSpec = useMemo(
+    () => (onlySetValue ? spec?.properties?.[rowKey] : spec?.patternProperties?.['^.*$']),
```

Comment:
does all the spec has `patternProperties`?

Reply:
No. When the spec is `Type.Record(Type.String(), Type.Any())`, it has the `patternProperties`. But it doesn't matter if it doesn't have the `patternProperties`, because the value's input widget would be the expression widget.

Reply:
I mean is this a typebox implementation or a general JSON-schema implementation?

Reply:
The `patternProperties` is a general JSON-schema implementation.

![image](https://user-images.githubusercontent.com/25414733/182057782-1dc370f2-aaac-437c-8c13-6687daffc90b.png)